apps/condo/components/LinkWithIcon.tsx

```diff
@@ -45,12 +49,10 @@ export const LinkWithIcon = (props: ILinkWithIconProps) => {
 
     return (
         <StyledButton type='link' onClick={handleClick}>
-            <Space size={16}>
-                {props.icon && <IconContainer className='icon'>{props.icon}</IconContainer>}
-                <Typography.Text className='text' style={{ fontSize: '12px' }}>
-                    {intl.formatMessage({ id: props.locale })}
-                </Typography.Text>
-            </Space>
+            {props.icon && <IconContainer className='icon'>{props.icon}</IconContainer>}
```

Comment:
You need to remove Space from imports. Antd style assumes to use the Space component here